bindings/core/src/response.rs

```diff
@@ -300,13 +300,15 @@ pub enum Response {
     /// Response for:
     /// - [`ClaimableOutputs`](crate::method::WalletMethod::ClaimableOutputs)
     OutputIds(Vec<OutputId>),
+    // TODO: update bindings in a separate PR
     /// Response for:
     /// - [`GetOutput`](crate::method::WalletMethod::GetOutput)
-    OutputData(Option<Box<OutputData>>),
+    OutputData(Option<Box<OutputWithExtendedMetadata>>),
+    // TODO: update bindings in a separate PR
```

Comment:
With the rename or what?

Reply:
yup

Reply:
The Python `OutputData` has more fields than the Rust version used to. Did we forget to keep those types in sync?

Reply:
Yes

Reply:
Updated the bindings